### PR TITLE
Fix rect-pth-pad hole computation

### PR DESCRIPTION
**Pull request by m-hilgendorf**

### CODE REVIEW

**Comment by d-haldane** [comment]
Looks like this function will have the same problem:

```
public defn general-round-rect-pth-pad (drill-r:Double, pad-w:Double, pad-h:Double, pad-r:Tuple<Double>) :
  general-round-rect-pth-pad(C, drill-r, drill-r, pad-w, pad-h, pad-r)
```

**Comment by m-hilgendorf** [comment]
Good catch. I'll audit this class of generator.

File: tests/test-pth-pads.stanza [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      fn (r:Double, w:Double, h:Double):
+        general-chamfered-rect-pth-pad(r, w, h, 0.5, CornerModifierAll)
+    ]
+    for (gen in generators, i in 1 to false) do: 
+      val ref = IndexRef(Ref("p"), i)
+      val def = gen(radius, width, length)
+      pad (ref): def at loc(0.0, to-double(i) * length + 0.5 * length)
+      
+      val cutout = find!({specifier(_) is Cutout}, layers(def))
+      #EXPECT(shape(cutout) is Capsule)
```
Comment (bpellenard) [inline tests/test-pth-pads.stanza:39]: `#ASSERT`?

**Review by bpellenard — DISMISSED** [review]

File: tests/test-pth-pads.stanza (hunk 1 above)
Comment (m-hilgendorf) [reply tests/test-pth-pads.stanza:39]: good call!

**Review by bpellenard — APPROVED** [review]

**Review by CuppoJava — APPROVED** [review]
Looks good to me.

**Pull request merged**